server.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express')
+const app = express()
+
+
+//routes
+
+app.get('/', (req, res) => {
+    res.sendStatus("Hello NODE API")
```

Comment:
The response from the root route should be sent using `res.send()` instead of `res.sendStatus()`. The latter is used to set the status of the response and does not send any content. [important]